src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1533,6 +1550,8 @@ internal partial class BoundQueryClause
     {
         protected override OperationKind ExpressionKind => OperationKind.None;
 
+        protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.Value);
```

Comment:
>protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.Value); [](start = 8, length = 104)

I am not sure if we want to expose the entire value here. I think it will include calls to the query operators, but at the design meeting I didn't get an impression that we would want to expose them tight now.